buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -56,6 +56,17 @@ internal class YamlWriter {
         appendln("  - $file")
       }
     }
+    val testApks = config.additionalTestApks.flatMap { (debugApk, instrumentationApks) ->
```

Comment:
This code can be somewhat simplified a bit since the instrumentation apk will take whatever value is above is as the debug apk. See the readme. https://github.com/TestArmada/flank/blob/master/README.md
```
if (config.additionalTestApks.isNotEmpty()) {
  appendln("  additional-app-test-apks:")
  config.additionalTestApks.entries.forEach { debugApk, instrumentationApks ->
    appendln("    - app: ${debugApk}")
    instrumentationApks.forEach {
        appendln("    test: ${it}")
    }
  }
```
On the other hand, I do like the more explicit output format that you created here. Let me think this one over overnight. :)

Reply:
Should I change this?